python/paddle/fluid/tests/unittests/test_cast_op.py

```diff
@@ -69,5 +71,17 @@ def test_check_output(self):
         self.check_output(atol=1e-3)
 
 
+class TestCastOpError(op_test.OpTest):
+    def test_errors(self):
+        with program_guard(Program(), Program()):
+            # The input type of cast_op must be Variable.
+            x1 = fluid.create_lod_tensor(
+                np.array([[-1]]), [[1]], fluid.CPUPlace())
+            self.assertRaises(TypeError, fluid.layers.cast, x1)
+            # The input dtype of softmax_op must be bool, float16, float32, float64, int32, int64, uint8.
```

Comment:
The input dtype of softmax_op->The input dtype of cast_op, a typo

Reply:
Done